src/main/kotlin/net/maizegenetics/phgv2/cli/SetupEnvironment.kt

```diff
@@ -0,0 +1,62 @@
+package net.maizegenetics.phgv2.cli
```

Comment:
Is this the same at init-db?

Reply:
No.  Setting up the environment is creating a phgv2 conda environment.  Initdb is to create the initial tiledb arrays to which we'll store the gvcfs.

I will work on the initDb stuff today/tomorrow.